Source/WebKit/UIProcess/Inspector/win/RemoteWebInspectorUIProxyWin.cpp

```diff
@@ -131,9 +134,48 @@ WebPageProxy* RemoteWebInspectorUIProxy::platformCreateFrontendPageAndWindow()
     return m_webView->page();
 }
 
+void RemoteWebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&& saveDatas, bool forceSaveAs)
```

Comment:
Generally, `UNUSED_PARAM` is used only with `#if`. I think you should do `/* forceSaveAs */` in this case.